src/config.ts

```diff
@@ -205,3 +229,29 @@ export function getContentConfig({ BEE_API_URLS, REUPLOAD_PERIOD }: EnvironmentV
     refreshPeriod: Number(REUPLOAD_PERIOD),
   }
 }
+
+export function getFilteringConfig({ FILTERING_CONFIG_FILE }: EnvironmentVariables = {}): FilteringConfig {
+  const NO_FILTERING: FilteringConfig = { active: false, key: '', url: '', prompt: '' }
+
+  if (!FILTERING_CONFIG_FILE) {
+    //console.warn('FILTERING_CONFIG_FILE is not defined, filtering is disabled')
+
+    return NO_FILTERING
+  }
+
+  try {
+    const result = readFileSync(join(__dirname, FILTERING_CONFIG_FILE), 'utf-8')
+    const filteringConfig: FilteringConfig = JSON.parse(result)
+
+    return { ...filteringConfig, active: true }
+  } catch (error) {
+    /*
+    console.error(
```

Comment:
I think this error should be logged to see if there is a problem with the input parameter config.

Reply:
fix: https://github.com/Solar-Punk-Ltd/gateway-proxy/commit/1fee57478db5727aab604d2404dec348519b36de